src/test/index.ts

```diff
@@ -0,0 +1,32 @@
+import { config } from 'dotenv';
+import { setup } from '../index';
+import axios from 'axios';
+import { expect } from 'chai';
+
+before(async () => {
+  config({ path: `${process.cwd()}/test.env` });
+  await setup();
+});
+
+describe('console log', () => {
+  it('teste', async () => {
+    console.log('teste');
+  });
+});
```

Comment:
acho que esse exemplo não precisa ir pro codigo, imagino que era mais um teste pra vc ver o describe e it